integration_tests/jira_issues_tests.feature

```diff
@@ -20,8 +20,7 @@ Feature: Jira Issues Tests
      And d3a is installed
      And export is_needed
      When we run the simulation with setup file strategy_tests.home_cp_ess_load and parameters [24, 60, 60, 1]
-     Then there should be no unmatched loads
```

Comment:
I should have mentioned this earlier, sorry, but the check if the load received all the energy that it demanded is important for most of the times this stop was implemented. 
I doubt that there is a handy step function implemented that loops over all loads and checks if the energy demand was always fulfilled. 
It should not be that hard actually, because the `_energy_requirement_Wh` member of the load state is buffering this information.
TLDR;
Please replace `there should be no unmatched loads` with a to-be-implemented function `all load demand in setup was fulfilled on every market slot`
Thanks a lot!

Reply:
@hannesdiedrich thanks for reviewing, do you think that I should create a ticket for it too. ?

Reply:
No, this should be part of this ticket.
1. to not get forgotten or pushed back 
2. We need these checks, especially for the setup.jira integrationtests.
@spyrostz correct me if I am overly cautious.

Reply:
@hannesdiedrich Yes I meant a ticket for implementing the function

Reply:
@Hassan754 yes, i meant to do it in the frame of this ticket.

Reply:
I agree with @hannesdiedrich that even though we deprecate the unmatched loads endpoint, in the tests we still need to detect if a device did not consume as much energy as the test dictates it too, and I think it would be good to be done in the context of this story because IIRC this is a check in many tests.

Reply:
Done